### DISCUSSION TITLE
Thank you. Very useful extension

### DISCUSSION THREAD

**Discussion by le-khang** [bbc-mc/sdweb-merge-block-weighted-gui#35] [General]
It's awesome! I can merge models in many different ways and got fantastic results. Thank you very much.
The only thing that bothers me is that my merged model is too large, occupying 8.5GB when I combine two 5.2GB models together. Does anyone know how to reduce the size, please?

**Comment by bbc-mc** [comment]
Hi, Im grad to hear that.

Try this:
https://github.com/arenasys/stable-diffusion-webui-model-toolkit
this extension has feature to 'Cleaning/pruning models', which decrease model size in many case.

**Reply by le-khang** [reply]
Thank you, I'll try that. This extension is a lot of fun. I can quickly create 30-50 random formulas to combine 2 models, and some of them have exceeded my expectations and also my storage lol.

May I ask if you have any plans to add new features in the future? Like combine 3-4 files simultaneously instead of 2. Or, I would love to have the same functions available for Lora as well. I can make hundreads of Lora files without storage issue. It's really exciting!
